### DISCUSSION TITLE
Deactivate interactive reply message

### DISCUSSION THREAD

**Discussion by bilo2000** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#66] [General]
Hi
Is there any way to deactivate the interactive reply button message after customer made the selection
I dont want to allow him to use it again

**Comment by gabrieldwight** [comment]
The quick reply button message deactivates the button selected after a user has made a selection. The only way is to handle your webhook using the message id to check if the user has already replied and not process the message if the user has selected another reply button.
